src/pwncore/routes/ctf/start.py

```diff
@@ -121,8 +118,8 @@ async def start_docker_container(ctf_id: int, response: Response):
 
 @atomic()
 @router.post("/stopall")
-async def stopall_docker_container(response: Response):
-    team_id = get_team_id()  # From JWT
+async def stopall_docker_container(response: Response, jwt=Depends(get_jwt)):
```

Comment:
Look at this section and change the style in which dependency is defined

https://fastapi.tiangolo.com/tutorial/dependencies/#share-annotated-dependencies

Reply:
As mentioned here: https://github.com/lugvitc/pwncore/pull/6#discussion_r1404238694